misk-hibernate/src/main/kotlin/misk/hibernate/Session.kt

```diff
@@ -5,6 +5,9 @@ import kotlin.reflect.KClass
 
 interface Session {
   val hibernateSession: org.hibernate.Session
+  // Token to identify a session. Tokens stay the same across transaction retries.
```

Comment:
nice

Reply:
Yes but this should be a ktdoc, not a single line comment :-)

Reply:
👍 True!